script/Ciel/Day 8/08_01_CIEL08_1B.txt

```diff
@@ -348,25 +380,26 @@ The time is seven-thirty.
 [sha:f500d73590a2148796606414a1c6b80bd2a0d4e1]{
 -- Page 62, Offset 18416.
 -- そろそろ家を出ないと間に合わない。
-I have to leave soon or I won't make it in time.
+I'd better leave soon if I want to make it on time.
 }
 [sha:6c0c9e9c4c03dee2ba06c8f2f570cf0f0d409973]{
 -- Page 63, Offset 18417.
 -- 「……それはそれとして。
-"...That being said.
+"That being said...
 }
 [sha:c21ea503a983716d10a5b5cf3312700a404469df]{
 -- Page 63, Offset 18418.
 -- アルクェイドとの事は黙っているしかないんだよな……」
-I had no choice but to keep silent about Arcueid..."
+I'd better keep quiet about Arcueid."
 }
 [sha:95dc2b22837e969c83f4ad4253a366dff23c12d5]{
 -- Page 64, Offset 18419.
 -- 先輩とアイツが水と油なのは分かりきっているし。
-I clearly get that Senpai and her are like water and oil.
+I'm painfully aware that Arcueid and Senpai are like water and oil.
```

Comment:
Isn't it "oil and water"? Admittedly, a very pedantic line of thought.

Reply:
I legit spent so long going back and forth on this. I initially corrected it to oil and water, but then managed to convince myself it was the other way around again? Anyway, changing.